src/Components/AdministratorPage/PanelRemarks/PanelRemarks.jsx

```diff
@@ -203,7 +218,7 @@ const PanelRemarks = () => {
                       text: 'Panel Remarks',
                       value: 'REMARKS',
                       content: showPanelRemarkForm ? <PanelRemarksForm mode={mode} categories={categoryFilter} remarkData={editRemarkData} onClose={closeForm} /> :
-                        <PanelRemarksTable panelRemarks={panelRemarksData} fetchData={fetchAndSet} openForm={openForm} />,
+                        <PanelRemarksTable ref={tableRef} panelRemarks={panelRemarksData} fetchData={fetchAndSet} openForm={openForm} isPrintView={printViewLoading} />,
```

Comment:
```suggestion
                        <PanelRemarksTable ref={tableRef} panelRemarks={panelRemarksData} fetchData={fetchAndSet} openForm={openForm} />,
```

i dont think that's used, is it?

Reply:
done :)